space-cli/cmd/modules/cluster/getters.go

```diff
@@ -0,0 +1,67 @@
+package cluster
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/model"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils"
+	"github.com/spaceuptech/space-cloud/space-cli/cmd/utils/transport"
+)
+
+// GetClusterConfig gets clusters config
+func GetClusterConfig() ([]*model.SpecObject, error) {
+	url := fmt.Sprintf("/v1/config/cluster")
+
+	// Resp is the object returned by every handler to client
+	type Resp struct {
```

Comment:
```suggestion
	type resp struct {
```
Remove the comment or change it